frontend/src/components/MapTooltip/MapTooltip.tsx

```diff
@@ -10,43 +14,89 @@ type MapTooltipProps = {
   coordinates: LngLat
 }
 
-let MapTooltip: React.StatelessComponent<MapTooltipProps> = ({ children, coordinates }) =>
-  <Popup
-    anchor='bottom'
-    className='MapTooltip'
-    coordinates={coordinates.toArray()}
-    offset={[0, -20]}
-  >
-    <table>
-      <tbody>
-        {children}
-      </tbody>
-    </table>
-  </Popup>
-
-type Props = {
+let MapTooltip: React.StatelessComponent<MapTooltipProps> = ({ children, coordinates }) => {
+  return (
+    <Popup
+      anchor='bottom'
+      className='MapTooltip'
+      coordinates={coordinates.toArray()}
+      offset={[0, -20]}
+    >
+      {children}
+    </Popup>
+  )
+}
+
+type ProviderProps = {
+  store: Store
+  point: GeoJSONEventData
+}
+
+type RepresentativePointProps = {
   point: GeoJSONEventData
 }
 
+function _ToggleNextProvider(store: Store, maxIndex: number) {
+  console.log('toggle next')
+  let index = store.get('providerIndex')
+  if (store.get('providerIndex') === maxIndex)
+    return
+  store.set('providerIndex')(index + 1)
+}
+
+function _TogglePrevProvider(store: Store) {
```

Comment:
[nit] We should add a lint rule for this, but try to keep function names camelCased.

Reply:
👍